cfspopcon/formulas/plasma_profiles/plasma_profiles.py

```diff
@@ -42,11 +42,13 @@ def calc_peaked_profiles(
     dilution: Unitfull,
     beta_toroidal: Unitfull,
     normalized_inverse_temp_scale_length: Unitfull,
+    profile_form: Optional[ProfileForm] = None,
+    density_profile_form: Optional[ProfileForm] = None,
+    temp_profile_form: Optional[ProfileForm] = None,
```

Comment:
How about replacing `profile_form` fully and requiring `density_profile_form` and `temp_profile_form`? 

I feel like that might be easier than maintaining both options and removes the possibility for users specifying both and getting an error. WDYT?

Reply:
Good idea. I've changed this now.